Cannot deploy ansible service broker because of missing docker image tag

## Bug:

**What happened**:
Error when deploying ansible-service-broker from simple-broker-template
Caused by non existing docker images with `sprint142` tag:
https://github.com/openshift/ansible-service-broker/blob/master/templates/simple-broker-template.yaml#L149

**What you expected to happen**:
Broker deployed successfully

**How to reproduce it**:
```
oc new-project ansible-service-broker
curl -s https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/simple-broker-template.yaml | oc process -n "ansible-service-broker" -f - | oc create -f -
```

Tag `latest` fixed the issue

Yeah @jmrodri, the template deploys just fine. I think the issue is the broker pod is failing. I was able to recreate the failure myself that @psturc is seeing [here](https://gist.github.com/psturc/f68de547fcd65a720156a265345ef778). My log snippet:

```
[2018-04-10T14:03:27.781Z] [DEBUG] - Creating k8s apiserver
[2018-04-10T14:03:27.785Z] [ERROR] - Unable to retrieve cluster roles rules from cluster
 You must be using OpenShift 3.7 to use the User rules check.
the server could not find the requested resource
```

If I recall correctly, this is related to `auto_escalate` in the template.

--- a/templates/simple-broker-template.yaml
+++ b/templates/simple-broker-template.yaml
@@ -146,7 +146,7 @@ objects:
       spec:
         serviceAccount: asb
         containers:
-        - image: ansibleplaybookbundle/origin-ansible-service-broker:sprint142
+        - image: ansibleplaybookbundle/origin-ansible-service-broker:release-1.1
           name: asb
           imagePullPolicy: IfNotPresent
           volumeMounts:
@@ -389,7 +389,7 @@ parameters:
 - description: APB Image Tag
   displayname: APB Image Tag
   name: TAG
-  value: sprint142 
+  value: release-1.1
 
 - description: OpenShift User Password
   displayname: OpenShift User Password

